src/base_DAO.mligo

```diff
@@ -40,8 +38,7 @@ let allowing_xtz (param, store, config : allow_xtz_params * storage * config) =
   | Accept_ownership           -> accept_ownership(store)
 
 (*
- * The actual DAO contract, which in this version is the same independently from
- * the DAO logic.
+ * The actual DAO contract, which is the same independently from the DAO logic.
```

Comment:
>  * The actual DAO contract, which is the same independently from the DAO logic.

May be we can further rephrase it as "which is the same, despite the specific DAO logic included", or something simlar.